contracts/truefi2/FixedTermLoan.sol

```diff
@@ -221,6 +221,9 @@ contract FixedTermLoan is IFixedTermLoan, ERC20 {
         uint256 tokenRedeemAmount = loanRedeemAmount.mul(_tokenBalance()).div(_totalSupply);
         tokenRedeemed = tokenRedeemed.add(tokenRedeemAmount);
 
+        if (address(ftlAgency) == msg.sender && status == Status.Settled) {
+            borrowingMutex.unlock(borrower);
+        }
```

Comment:
Can we check instead if `borrowingMutex.locker(borrower) == msg.sender && ...`? Then we wouldn't need to keep `ftlAgency` in storage.

Reply:
`borrowingMutex.locker(borrower)` is loan token aka FixedTermLoan

Reply:
Yeah, my bad.